OpenSim/Common/ComponentSocket.h

```diff
@@ -689,60 +694,60 @@ class  Input : public AbstractInput {
     SimTK::ResetOnCopy<AliasList> _aliases;
 }; // END class Input<Y>
         
-/// @name Creating Connectors to other objects for your Component
+/// @name Creating Sockets to other objects for your Component
```

Comment:
A Socket isn't restricted to OpenSim::Objects, so perhaps avoid the word "objects" here and just say "Creating Sockets to other Components"?